apps/design/shared/src/layout.ts

```diff
@@ -767,18 +1121,23 @@ function layOutBallotHelper(
   const precinctIndex = election.precincts.findIndex(
     (p) => p.id === precinct.id
   );
-  const contests = getContests({ election, ballotStyle }).filter(
-    // TODO: support ballot measures
-    (contest) => contest.type === 'candidate'
-  );
+  // For now, just one section for candidate contests, one for ballot measures.
+  // TODO support arbitrarily defined sections
+  const contests = getContests({ election, ballotStyle });
+  if (contests.length === 0) {
+    throw new Error('No contests assigned to this precinct.');
+  }
+  const contestSections: Array<AnyContest[]> = iter(contests)
```

Comment:
Part of the challenge of understanding what `ContestSection` was, was that here it means something different. I think that there needs to be more clarity in the code between the two different concepts:

- _semantic_ contest sections, which currently are only "candidate" and "ballot measure" but later will be more
- _layout_ contest sections, which are confined to a single page. although a single page can have multiple layout contest sections

These are not the best names, though, and I'm wondering what would be better. Even something like `ContestBlock` vs `contestSection` would be helpful if it were consistent. "Block" could instead be "Pane". Or maybe it could be called `ContestColumns`. I think "section" is good to reserve for the semantic sense and we need something else for the layout concept.

Reply:
Great callout! Will change.

Reply:
Considering that you're planning to rewrite the layout module, this doesn't seem as important to me unless a lot of this code will live on.